ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -580,5 +579,9 @@ Clean Up Test Project
     ...    namespace=${test_ns}
     ${rc}    ${out}=    Run And Return Rc And Output    oc delete project ${test_ns}
     Should Be Equal As Integers    ${rc}    ${0}
-    ${rc}    ${out}=    Run And Return Rc And Output    oc wait --for=delete namespace ${test_ns} --timeout=300s
-    Should Be Equal As Integers    ${rc}    ${0}
+    IF    ${wait_prj_deletion}
```

Comment:
There are a lot of keywords deleting projects, for example `Delete Data Science Project From CLI` in ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

Could you consider enhancing the existing one for your purposes in another PR?

Reply:
I thought about that while reviewing the code. I think the main difference is that the first one you mention handled DS Projects, while in this case we're handling basic OCP projects. 

I didn't apply enhancements for now, need to thinki a bit more about how to implement it, but I agree